main.go

```diff
@@ -16,13 +16,14 @@ import (
 )
 
 var (
-	authToken = kingpin.Flag("auth", "Auth token").String()
-	team      = kingpin.Flag("team", "Team").Required().String()
-	pdTeam    = kingpin.Flag("pd-team", "Team in PagerDuty if different from Team").String()
-	since     = kingpin.Flag("since", "Since date/time").Required().String()
-	until     = kingpin.Flag("until", "Until date/time").Required().String()
-	urgency   = kingpin.Flag("urgency", "Urgency").Default("high").String()
-	replace   = kingpin.Flag("replace", "Replace titles with regex").Strings()
+	authToken  = kingpin.Flag("auth", "Auth token").String()
+	team       = kingpin.Flag("team", "Team").Required().String()
+	pdTeam     = kingpin.Flag("pd-team", "Team in PagerDuty if different from Team").String()
+	since      = kingpin.Flag("since", "Since date/time").Required().String()
+	until      = kingpin.Flag("until", "Until date/time").Required().String()
+	urgency    = kingpin.Flag("urgency", "Urgency").Default("high").String()
+	replace    = kingpin.Flag("replace", "Replace titles with regex").Strings()
+	tagFilters = kingpin.Flag("tags", "Filter PagerDuty incidents by Datadog tags").Strings()
```

Comment:
Mind adding this as an example to the `README`? Thank you 🙇

Reply:
I added it to the README, but please feel free to suggest a more specific example if you have one in mind 🙂